ddl/partition.go

```diff
@@ -271,15 +271,15 @@ func checkAddPartitionTooManyPartitions(piDefs int) error {
 	return nil
 }
 
-func getPartitionIDs(table *model.TableInfo) []int64 {
+func getPhysicalTableIDs(table *model.TableInfo) []int64 {
```

Comment:
Maybe we need not change its name, because it does not return the table id when the table is not partitioned.

Reply:
Keep the previous name `getPartitionIDs `?